plugins/BEdita/Core/tests/TestCase/Model/Behavior/HistoryBehaviorTest.php

```diff
@@ -0,0 +1,251 @@
+<?php
+/**
+ * BEdita, API-first content management framework
+ * Copyright 2019 ChannelWeb Srl, Chialab Srl
+ *
+ * This file is part of BEdita: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * See LICENSE.LGPL or <http://gnu.org/licenses/lgpl-3.0.html> for more details.
+ */
+
+namespace BEdita\Core\Test\TestCase\Model\Behavior;
+
+use BEdita\Core\Utility\LoggedUser;
+use Cake\Core\Configure;
+use Cake\ORM\TableRegistry;
+use Cake\TestSuite\TestCase;
+
+/**
+ * @coversDefaultClass \BEdita\Core\Model\Behavior\HistoryBehavior
+ */
+class HistoryBehaviorTest extends TestCase
+{
+    /**
+     * Fixtures
+     *
+     * @var array
+     */
+    public $fixtures = [
+        'plugin.BEdita/Core.Applications',
+        'plugin.BEdita/Core.ObjectTypes',
+        'plugin.BEdita/Core.Properties',
+        'plugin.BEdita/Core.PropertyTypes',
+        'plugin.BEdita/Core.Relations',
+        'plugin.BEdita/Core.RelationTypes',
+        'plugin.BEdita/Core.Objects',
+        'plugin.BEdita/Core.Profiles',
+        'plugin.BEdita/Core.Users',
+        'plugin.BEdita/Core.ObjectRelations',
+        'plugin.BEdita/Core.Trees',
+        'plugin.BEdita/Core.History',
+    ];
+
+    /**
+     * {@inheritDoc}
+     */
+    public function setUp()
+    {
+        parent::setUp();
+        LoggedUser::setUser(['id' => 1]);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function tearDown()
+    {
+        parent::tearDown();
+        LoggedUser::resetUser();
+    }
+
+    /**
+     * Test `initialize` method
+     *
+     * @covers ::initialize()
+     */
+    public function testInitialize()
+    {
+        $Documents = TableRegistry::get('Documents');
```

Comment:
Ok with `Documents`. But it seems that with `Users`, `Profiles` and other types, history doesn't work.
We could add one of those types in tests.

Reply:
Solved now, tests have been updated.
Solution far from optimal, a model table refactoring is necessary. Will be done in a new issue.